src/protocols/DRMLease.cpp

```diff
@@ -225,7 +225,7 @@ CDRMLeaseDevice::CDRMLeaseDevice(SP<Aquamarine::CDRMBackend> drmBackend) : backe
     auto fd = drm->getNonMasterFD();
 
     if (fd < 0) {
-        LOGM(ERR, "Failed to dup fd for drm node {}", drm->gpuName);
+        Debug::log(ERR, "DRMLease Failed to dup fd for drm node {}", drm->gpuName);
```

Comment:
Note that it does not preserve format in which `DRMLease` is in square brackets. I guess there's no need to use `Debug::log` directly but just `protoLog` without `LOGM` indirection.

Reply:
Should just refactor the entire logging in these protocols heh

Reply:
added the brackets back but yeah a logging refactoring is needed in the protocols so this doesnt accidently occur again, but its out of the scope of this PR. is like 40 different files with various places it needs a checkover.